src/lib/templateengine.js

```diff
@@ -274,33 +276,20 @@ function TemplateEngine( undefined ) {
   
   this.enableAddressQueue = $.getUrlVar('enableQueue') ? true : false;
   
-  this.backend = 'cgi-bin'; // default path to backend
+  this.backend = 'default';
+  this.backendUrl = null;
```

Comment:
In this case I want to intialize a variable with no value set, what is exactly what 'null' is used for (also in Javascript).
So this.backendUrl === null is an expected (because set by user) behaviour, whereas this.backendUrl === undefined would mean something went terribly wrong. To distinguish these cases the initialization of variables with 'null' is the way to go from my point of view.